### PR TITLE
Adds homepage styling, fontawesome and 2 fonts

### PR DESCRIPTION
**Pull request by skuhlmann**

### CODE REVIEW

File: app/styles/app.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  height: 80px;
+  margin-bottom: 90px;
+}
+
+#home-headline {
+  text-align: center;
+}
+
+#home-headline h1 {
+  font-family: 'Open Sans', sans-serif;
```
Comment (chandracarney) [inline app/styles/app.css]: I think this font-family line can come off since it's defined on line 13

**Comment by chandracarney** [comment]
Looks good minus one small CSS note!

**Comment by trayo** [comment]
This is the build error message that it's failing on: `Path or pattern "bower_components/font-awesome/css/font-awesome.css" did not match any files`.

I assume it's because the font needs to be in that folder or it's not installed correctly? It might be possible to install it with some addon at http://www.emberaddons.com/

File: app/styles/app.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
+
+@import url(http://fonts.googleapis.com/css?family=Open+Sans:400,800,700);@import url(http://fonts.googleapis.com/css?family=Gentium+Basic:400,700);
```
Comment (trayo) [inline app/styles/app.css]: These http links will break on Heroku, they need to be downloaded or linked to with `https`. Also, I thought `@import` was a method that is for sass only?

File: app/styles/app.css (hunk 1 above)
Comment (skuhlmann) [inline app/styles/app.css]: Removed this line. Thanks.

File: app/styles/app.css (hunk 2 above)
Comment (skuhlmann) [inline app/styles/app.css]: @import can be used in css. 

I think we should try this on heroku. I have used these in the past on app and they are working fine deployed on heroku,.

**Comment by skuhlmann** [comment]
Regarding "bower_components/font-awesome/css/font-awesome.css"...

This is strange because the css file is in there in that exact location... i think there was an issues with the bower.json dependency. I've updated and will push again.

**Comment by skuhlmann** [comment]
Sweet. It passed this time.

**Comment by trayo** [comment]
Nice! :sparkles:

**Pull request merged**